pkg/cmdlets/prompts/TOS.go

```diff
@@ -0,0 +1,97 @@
+package prompts
+
+import (
+	"io"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/ActiveState/cli/internal/constants"
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/internal/prompt"
+)
+
+type TOS interface {
+	GetTOSText() (string, error)
+}
+
+type onlineTOS struct {
+	TOS
+}
+
+func NewOnlineTOS() *onlineTOS {
+	return &onlineTOS{}
+}
+
+func (tos *onlineTOS) GetTOSText() (string, error) {
+	resp, err := http.Get(constants.TermsOfServiceURLText)
+	if err != nil {
+		return "", errs.Wrap(err, "Failed to download the Terms Of Service document.")
+	}
+	if resp.StatusCode != http.StatusOK {
+		return "", errs.New("The server responded with status '%s' when trying to download the Terms Of Service document.", resp.Status)
+	}
+	defer resp.Body.Close()
+
+	tosText := new(strings.Builder)
+	_, err = io.Copy(tosText, resp.Body)
+	if err != nil {
+		return "", errs.Wrap(err, "Failed to read Terms Of Service contents.")
+	}
+
+	return tosText.String(), nil
+}
+
+type offlineFileTOS struct {
+	TOS
+	tosFilePath string
+}
+
+func NewOfflineFileTOS(tosFilePath string) *offlineFileTOS {
+	return &offlineFileTOS{tosFilePath: tosFilePath}
+}
+
+func (tos *offlineFileTOS) GetTOSText() (string, error) {
+	b, err := os.ReadFile(tos.tosFilePath)
```

Comment:
This path only exists on dev machines. Once built and shipped it won't know how to find this on users computers.

Instead use embed: https://pkg.go.dev/embed